pages.it/common/dash.md

```diff
@@ -0,0 +1,28 @@
+# dash
+
+> Debian Almquist Shell, una implementazione di `sh` moderna, che conforme a POSIX, (non compatibile con Bash).
+> Maggiori informazioni: <https://manned.org/dash>.
+
+- Avvia una sessione shell interattiva:
+
+`dash`
+
+- Esegui un comando, ed esci subito:
+
+`dash -c "{{comando}}"`
+
+- Esegui un script:
+
+`dash {{percorsi/al/script.sh}}`
+
+- Esegui comandi da un script, stampando ogni comando prima di eseguirlo:
+
+`dash -x {{path/to/script.sh}}`
```

Comment:
I didn't add all the suggestions for `path/to` because I wasn't sure about the translation:
```suggestion
`dash -x {{percorsi/al/script.sh}}`
```